tests/test_photo.py

```diff
@@ -63,7 +63,7 @@ class TestPhoto:
     width = 800
     height = 800
     caption = '<b>PhotoTest</b> - *Caption*'
-    photo_file_url = 'https://python-telegram-bot.org/static/testfiles/telegram_new.jpg'
+    photo_file_url = 'https://python-telegram-bot.org/static/testfiles/telegram.jpg'
```

Comment:
why?

Reply:
because iisc that's the file the `photo` fixture uses. I don't know what exactly the difference is, but it was a first guess to fix the tests and at least didn't make them worse …

Reply:
welp, okay.